package.json

```diff
@@ -4,9 +4,11 @@
   "description": "Listen to YouTube videos, without the distracting visuals.",
   "main": "tests.js",
   "scripts": {
-    "lint:js": "jshint js test",
+    "lint:js": "eslint js/**",
```

Comment:
To be honest I don't know why but documentation mentions this :/. I'll correct it to `*.js`.